### PR TITLE
Add DESeq2 and limma model to output

### PR DESCRIPTION
**Pull request by WackerO**
This PR adds an output file to DESeq2 and limma that contains the model that was used

## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by pinin4fjords — CHANGES_REQUESTED** [review]
Looking good- but you need to resolve whatever git issue is causing you to hit other modules.

**Comment by WackerO** [comment]
Alright, did that. But now the linting is failing again, can we ignore that? It's because those other two modules have incorrectly written environment.ymls

**Comment by pinin4fjords** [comment]
Could you maybe do the limma one at the same time and save us a PR?

File: modules/nf-core/limma/differential/meta.yml [hunk 1]
```diff
@@ -63,6 +63,10 @@ output:
       type: file
       description: Serialised MArrayLM object
       pattern: "*.MArrayLM.limma.rds"
+  - model:
+      type: file
+      description: TXT-format limma model
+      pattern: "limma.model.tsv"
```
Comment (pinin4fjords) [inline modules/nf-core/limma/differential/meta.yml:69]:
```suggestion
      pattern: "*.limma.model.tsv"
```

Comment (pinin4fjords) [reply modules/nf-core/limma/differential/meta.yml:69]: Just for consistency with the other entries here

**Review by pinin4fjords — APPROVED** [review]
Looks good- thanks Oskar

**Pull request merged**